### DISCUSSION TITLE
Was / will the `/server-info/` API endpoint [be] removed?

### DISCUSSION THREAD

**Discussion by shamoon** [immich-app/immich#13329] [Q&A] [answered]
http://gethomepage.dev dev here, Im trying to track down a report about our immich 'widget'. tl;dr, trying to determine if the `/server-info/` API endpoint has truly been removed (yet?):

- Based on https://github.com/immich-app/immich/pull/10677 --> https://github.com/immich-app/immich/pull/11923 which I believe was released in https://github.com/immich-app/immich/releases/tag/v1.113.0 it looks like `/server/` was released but I'm unclear if the old one was removed.
- In the current demo (1.117) and a local install, it still works fine for me
- A user is reporting it fails on their install at the same version: https://github.com/gethomepage/homepage/discussions/4100#discussioncomment-10898068

What am I missing?

Finally, if the plan is to remove the old one, would you consider leaving a redirect of some kind to minimize the impact of this?

Thanks!

**Comment by alextran1502** [comment, marked as answer]
In the next version, `v1.118,0`, the old endpoint `/server-info` will be removed. The new endpoint `/server/` has been in the app for the last couple of releases since `v1.113.0`, so you can change it.

**Reply by shamoon** [reply]
Appreciate the answer. I suppose your `main` docker image doesn't really make it apparent to users that they're running a different version than the release.

One last note, again, I'd vote for keeping a redirect. I know this project is under "very active development" but with 208 releases in the bank surely there are others who rely on the API at this point and I can't say that the benefits of a semantic change to an API endpoint outweigh the downsides of a breaking change.

Of course, just my opinion. Thanks again.
